src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+/**
+ * Represents a To-do Task.
+ *
+ * @author Denzel Tan
+ */
+public class Todo extends Task {
+    protected String by;
```

Comment:
Perhaps you can comment or use JavaDocs to explain what is the use of this global variable 'by' as it seems you are not really using this class member within the class!